packages/uiweb/src/lib/components/space/SpaceBanner/SpaceBanner.tsx

```diff
@@ -1,15 +1,264 @@
 import React from 'react';
 import { useSpaceData } from '../../../hooks';
+import styled from 'styled-components';
 
 export interface ISpaceBannerProps {
   // Add props specific to the SpaceBanner component
-  temporary?: string; // just to remove eslint error of empty prop
+  title: string;
+  peopleInSpace: string[];
+  participants: number;
+  time: Date;
+  status: 'live' | 'scheduled' | 'ended';
+  host?: object;
+  theme?: object;
+  orientation?: 'maximized' | 'minimized';
 }
 
-export const SpaceBanner: React.FC<ISpaceBannerProps> = () => {
+export const SpaceBanner: React.FC<ISpaceBannerProps> = ({
+  title,
+  peopleInSpace,
+  participants,
+  time,
+  status,
+  host,
+  theme,
+  orientation,
+}) => {
   const { spaceBannerData, setSpaceBannerData } = useSpaceData();
 
   // Use spaceBannerData and setSpaceBannerData in your component
 
-  return <div>{/* Render your component content here */}</div>;
+  return (
+    <div>
+      <Container orientation={orientation}>
+        <ProfileContainer orientation={orientation}>
+          <PfpContainer>
+            <Pfp src="" alt="pfp" />
+          </PfpContainer>
+          <HostContainer>
+            <HostName>
+              
+              <Host>Host</Host>
+            </HostName>
+            <HostHandle></HostHandle>
+          </HostContainer>
+        </ProfileContainer>
+        {orientation === 'maximized' ? null : <Icon src="" />}
+        <Title orientation={orientation}>
+          {orientation === 'minimized' ? `${title?.slice(0, 30)}...` : title}
+        </Title>
+        <Status>
+          <Time orientation={orientation}>
+            <Icon />
+            <TimeText></TimeText>
+          </Time>
+          <Participants>
+            <ParticipantsIconContainer orientation={orientation}>
+              <ParticipantsIcon1 src="" alt="pfp"/>
+              <ParticipantsIcon2 src="" alt="pfp"/>
+              <ParticipantsIcon3 src="" alt="pfp" orientation={orientation} />
```

Comment:
since this data will be provided by the backend, in the form of an array, this should be implemented as an Participants.map() method in the UI.

The position of the images can be controlled by using absolute positioning and z-index instead of a CSS grid.

The orientation prop can be then used to determine whether to map 3 elements or 2 elements.